tests/interactive.py

```diff
@@ -6,7 +6,8 @@
 
 from __future__ import absolute_import
 import looper
-from . import conftest
+#from . import conftest
+import conftest
```

Comment:
@vreuter This change is unrelated; but it was required -- is it because of changes way we're doing the 3 to 2 thing now?